packages/noports_core/lib/src/sshnpd/sshnpd_params.dart

```diff
@@ -1,68 +1,82 @@
 import 'package:args/args.dart';
 import 'package:noports_core/src/common/default_args.dart';
-import 'package:noports_core/src/common/supported_ssh_clients.dart';
-import 'package:noports_core/src/common/utils.dart';
+import 'package:noports_core/src/common/file_system_utils.dart';
+import 'package:noports_core/src/common/types.dart';
+import 'package:noports_core/src/common/validation_utils.dart';
 
 class SSHNPDParams {
-  late final String device;
-  late final String username;
-  late final String homeDirectory;
-  late final String managerAtsign;
-  late final String atKeysFilePath;
-  late final String sendSshPublicKey;
-  late final String deviceAtsign;
-  late final bool verbose;
-  late final bool makeDeviceInfoVisible;
-  late final bool addSshPublicKeys;
-  late final SupportedSshClient sshClient;
-  late final String rootDomain;
-  late final int localSshdPort;
-  late final String ephemeralPermissions;
-  late final bool rsa;
+  final String device;
+  final String username;
+  final String homeDirectory;
+  final String managerAtsign;
+  final String atKeysFilePath;
+  final String deviceAtsign;
+  final bool verbose;
+  final bool makeDeviceInfoVisible;
+  final bool addSshPublicKeys;
+  final SupportedSshClient sshClient;
+  final String rootDomain;
+  final int localSshdPort;
+  final String ephemeralPermissions;
+  final SupportedSSHAlgorithm sshAlgorithm;
```

Comment:
Remove late from all of these, which means I had to change how they get set below